test/PCMacro/didset.swift

```diff
@@ -17,7 +17,7 @@ import PlaygroundSupport
 #sourceLocation(file: "main.swift", line: 10)
 struct S {
     var a : [Int] = [] {
-        didSet {
+        didSet(oldValue) {
```

Comment:
I had to add this otherwise the output below had a slight change in order - I don't think its correct but I also don't know what this "PCMacro" test does or how it works. Any clues?